app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/pocket/PocketStoriesComposables.kt

```diff
@@ -0,0 +1,285 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
From a file naming convention for Jetpack compose, I am wondering if we do feel the need to add the `Composables` suffix. Looking at the existing compose sample apps written, lead me to conclude that we don't really need to care about such differentiation. In Focus, I opted to just name it `TopSites.kt` to match the top level compose function that we want to be used to set the content. In this case, this would be `PocketStories.kt`.

Reply:
I agree that a file containing Composables shouln't by default end in `Composable`.
In this case though I went with this naming structure since I saw other files in the `viewholders` package ending in `View`/`Holder` so I wanted to make it a bit more clear about what may be found in this file - just `PocketStories` would not really say much in my opinion.
Can change if required.

Reply:
In the `AwesomeBar()` code I started to move most composables to separate files and use packages to organize them.